Data/CMakeLists.txt

```diff
@@ -58,7 +58,7 @@ endif(ENABLE_DATA_MYSQL)
 if(ENABLE_DATA_POSTGRESQL)
     find_package(PostgreSQL)
     if(POSTGRESQL_FOUND)
```

Comment:
I'm with you now (finally hehe) and yes,  it could probably be changed to adhere to the documentation but since it didn't break any functionality I didn't even notice it. The do however set PostgreSQL_FOUND first on line 178 and the set it to the value of POSTGRESQL_FOUND so functionalitywise it doesn't matter really. It doesn't follow the documentation exactly but since all other variables in Data/CMakeLists.txt are using uppercase only I think it looks more coherent using the "inofficial" POSGRESQL_FOUND?